src/noTemplateCallExpressionRule.ts

```diff
@@ -0,0 +1,48 @@
+import * as Lint from 'tslint';
+import * as ts from 'typescript';
+import { NgWalker, NgWalkerConfig } from './angular/ngWalker';
+import { BasicTemplateAstVisitor  } from './angular/templates/basicTemplateAstVisitor';
+import { RecursiveAngularExpressionVisitor  } from './angular/templates/recursiveAngularExpressionVisitor';
+import * as e from '@angular/compiler/src/expression_parser/ast';
+
+export class Rule extends Lint.Rules.AbstractRule {
+  public static metadata: Lint.IRuleMetadata = {
+    ruleName: 'no-template-call-expression',
+    type: 'functionality',
+    description: `Call expressions are not allowed in templates except in event handler bindings.`,
+    rationale: `The change detector will call functions used in templates very often. Use an observable instead.`,
```

Comment:
We can use a string literal here instead of template strings. There are already existing rules where this should be fixed as well.